bus-mapping/src/evm/gas.rs

```diff
@@ -14,6 +14,8 @@ impl GasCost {
     pub const SLOW: Self = Self(10);
     /// Constant cost for ext step
     pub const EXT: Self = Self(20);
+    /// Constant cost for every additional word when expanding memory
+    pub const MEMORY: Self = Self(3);
```

Comment:
why not use existing FASTEST field which represents "3" ?

Reply:
Following the yellow paper here where these are separate constants. Even though they currently have the same value, changing one does not mean the other one needs to be changed as well, which could happen at some point.

Reply:
It seems like that this `gas.rs` file is not used, or I just missed something?

Reply:
I also don't know why that is currently the case. I think there was once a TODO to remove one of them, but I don't know why it is currently like this so I just update both.